### ISSUE TITLE
MS Teams with different webhook domain still not working

### ISSUE THREAD

**Issue by SmallhillCZ** [containrrr/watchtower#858]
**Describe the bug**

It is the same issue as https://github.com/containrrr/watchtower/issues/792, however still present despite closed.

When the MS Teams webhook contains custom domain in host, e.g. `https://MYCOMPANY.webhook.office.com/webhookb2/` Watchtower fails to send notifications and logs:

`Failed to send notification via shoutrrr (url=teams://https%3A:@MYCOMPANY.webhook.office.com):  failed to send notification to teams, response status code 404 Not Found`

**To Reproduce**
1. Set MS Teams webhook in the new format:
```
WATCHTOWER_NOTIFICATIONS: msteams
WATCHTOWER_NOTIFICATION_MSTEAMS_HOOK_URL: https://MYCOMPANY.webhook.office.com/webhookb2/...@.../IncomingWebhook/.../...
```
2. Restart watchtower
3. See error

**Expected behavior**
Notification is sent to MS Teams channel without error

**Environment**
Tried with Docker versions `latest` and `latest-dev`

**Comment by austin-engle** [comment]
I am experiencing the same issue as well.

**Comment by piksel** [comment]
It should be fixed as of `v1.2.1`

**Comment by austin-engle** [comment]
I can confirm this is working now for me
